isos/vicadmin/dashboard.html

```diff
@@ -0,0 +1,100 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+
+    <title>VCH Admin</title>
+
+    <!-- Bootstrap core CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+
+    <!-- Custom styles for this template-->
+    <style type="text/css">
+      .navbar {
+        border-radius:0;
+      }
+    </style>
+
+  </head>
+
+  <body>
+
+    <nav class="navbar navbar-inverse">
+      <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="#">vSphere Integrated Containers</a>
+        </div>
+        <div id="navbar">
+          <form class="navbar-form navbar-right">
+            <button type="submit" class="btn btn-primary">View on Github</button>
+          </form>
+        </div><!--/.navbar-collapse -->
+      </div>
+    </nav>
+
+    <div class="container">
+
+      <div class="starter-template">
+        <h1>[VCH NAME] Admin</h1>
+        <p>v {{.Version}}</p>
+      </div>
+      <div class="row">
+        <div class="col-md-6">
+          <div class="panel panel-default">
+    	  	  <div class="panel-body">
+    	    	  <h2>Status</h2>
+    	    	  <ul>
+      	    	  <li>Containers</li>
+      	    	  <li>Network Connectivity</li>
+      	    	  <li>Firewall</li>
+      	    	  <li>Virtual Container Host (VCH)</li>
+      	    	  <li>License</li>
+    	    	  </ul>
+    	  	  </div>
+    	    </div>
+          <div class="panel panel-default">
+    	  	  <div class="panel-body">
+    	    	  <h2>VCH Info</h2>
+    	    	  <p>Docker Endpoint: 183.28.75.125:2376</p>
+    	    	  <p>Storage: 420GB remaining</p>
```

Comment:
We are doing one section of vic-admin at a time, with an Epic for each. These hard coded values will be replaced with appropriate variables when we reach that section of our project which is slated for July 27th.